plugins/node/opentelemetry-instrumentation-pg/src/index.ts

```diff
@@ -17,3 +17,4 @@
 export * from './instrumentation';
 export * from './types';
 export * from './enums/AttributeNames';
+export * from './enums/SpanNames';
```

Comment:
Actually, I think we should not export it from index, as this would add it to the instrumentation public API which would make it a possible breaking change if we make changes in the future.

Reply:
Done